force-app/main/default/objects/External_Committee__c/listViews/All_Open_Meetings.listView-meta.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ListView xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>All_Open_Meetings</fullName>
+    <columns>NAME</columns>
+    <columns>CREATED_DATE</columns>
+    <columns>Closed_Status__c</columns>
+    <columns>End_Time__c</columns>
+    <filterScope>Everything</filterScope>
+    <filters>
+        <field>Status__c</field>
+        <operation>notEqual</operation>
+        <value>Closed</value>
+    </filters>
+    <label>All Open Meetings</label>
```

Comment:
@deepakmulamalla : There are no list view sharing settings for this new list view you created. But the All list view has sharing settings. Please confirm why this difference?

Reply:
It's done as part of ESA-1965 user story and "All open meetings" list view is accessed by all users now
Previously the list view is restricted to internal and now it's removed.